assets/js/components/layout-control/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { clamp, isNaN } from 'lodash';
+import { Fragment } from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { RangeControl } from '@wordpress/components';
+
+/**
+ * A combination of range controls for product grid layout settings.
+ */
+const LayoutControl = ( { columns, rows, setAttributes } ) => {
```

Comment:
Should this be called "GridLayoutControl" to be more specific?

Reply:
I was thinking that after I'd written this up, so yeah - I've changed it.